doc/getting_going.rst

```diff
@@ -71,8 +71,9 @@ higher than 1.5.2 should work but have not been tested.
 
 If pyparsing is not installed on your system you can install it from
 within Ubuntu using the software center (search for the
-"python-pyparsing" module in the software center and install). If you
-do not run Ubuntu you could follow the instructions here
+"python-pyparsing" module in the software center and install). It is
```

Comment:
remove the Ubuntu-specific install info and show how to install using pip. Again show system wide if admin rights, then user install only and finally show how to uninstall.